Classes/Domain/Repository/BookRepository.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Books\Domain\Repository;
+
+use TYPO3\CMS\Extbase\Persistence\Repository;
+
+class BookRepository extends Repository
```

Comment:
Please add some PHPDoc with a short description (and an `@author` if you'd like to take the credit).

Reply:
Not sure we need such a comment for classes that have no special functionality beyond the standard Extbase way. I'll add a comment anyways since this is an example extension for learners.